### DISCUSSION TITLE
Question about requirements for loaded series, diplaying 1 pane in window selectively, etc.

### DISCUSSION THREAD

**Discussion by sscotti** [Kitware/VolView#302] [Ideas]
I just started looking at this integrated with Orthanc via a "Plug-in" courtesy of the people at Orthanc and Osimis.

One user has mostly MR studies from a 0.25 T scanner, and a variety of series, some of which I think are isotropic MPR / "resl" in Esoate parlance.  Many of the others are just T1, T2, STIRS, etc. with only maybe 25 slices per series.  The MPR's have around 200 instances or more sometimes.  I have attached a screen shot of a sample for initially evaluating it, using just the raw data series with 286 instances:

<img width="1796" alt="image" src="https://user-images.githubusercontent.com/236933/231742436-ad6e54c5-a98f-4a90-bfdf-2e14190bd65f.png">

That seems to work quite nicely with the cross-hairs, etc.  Just wondering what the "requirements" are for uploaded series, etc.

Also, would it be possible to select a single view pane and have it display just that pane in the available space for the viewport ?

e.g. instead of 4 ports, show just 1 ?

Also related.  Are there 'pre-sets' for CT Bone, Abd, Liver Windows and other presets for W/L ?

https://radiopaedia.org/articles/windowing-ct

![image](https://user-images.githubusercontent.com/236933/231755982-4e6c9c6f-8ac9-4de5-911c-9c29e611691a.png)

**Comment by floryst** [comment]
> That seems to work quite nicely with the cross-hairs, etc. Just wondering what the "requirements" are for uploaded series, etc.

I'm not sure what you refer to when you say "requirements". Could you please specify what you are looking for here?

> Also, would it be possible to select a single view pane and have it display just that pane in the available space for the viewport ?

That will soon be available! It's a small change, but I've been tackling big-ticket items lately, so this will happen soon.

> Also related. Are there 'pre-sets' for CT Bone, Abd, Liver Windows and other presets for W/L ?

Linking https://github.com/Kitware/VolView/issues/181. We are targeting this functionality as well.

**Reply by sscotti** [reply]
Re:   "requirements":

I'll have to play around with it a little again with some sample studies.  However, it seems like the viewer is optimized for viewing CT and MRI series that were obtained as a '3D' volume acquisitions, not standard FSE T2 ?  Those have this tag:

**0018,0023 - MRAcquisitionType: 2D**

With the Orthanc integration it seems like it would be most effective to just load a single series, not the whole study, probably 3D axial or 'raw data', whatever that is, and nothing else.  That would save a little time also because only the series of interest would be loaded ??

As an example, I have an L-spine comprised of there series.  This scanner is a low-T 0.25 scanner, and it is a little limited because of the coils and the software.

0° - Sagittal SE T1 | MR | 13
0° - Sagittal FSE T2 | MR | 13
0° - Coronal HYCE MPR | MR | 59 ??
0° - Sagittal HYCE MPR | MR | 63 ??
**0° - Axial HYCE MPR | MR | 118**
0° - Sagittal STIR | MR | 13

In the VolView Viewer with that study I see 2 separate Axial HYCE MPR series listed, one with 45 and one with 73 instances.  Doing a little math, 45 + 73 = 118 probably is related to that Axial HYCE MPR being constructed from 2 separate acquisitions 'pasted' together in this case because I guess the coil could not cover the entire spine in one shot here.

So my conclusion is that the viewer is loading just the axial series.  Why not the other MPR's ?  Also, the viewer is quite nice, can it not also display the 2D acquisitions if one wants to see them ?

**0018,0023 - MRAcquisitionType: 3D**

and it is getting split because they were sort of 'glued' together by the tech before sending to PACS.

So one question would be, is it worth just sending the raw data, or send the raw data in addition to the MPR's created by the tech ?

I am using the Orthanc Plug-in.  It is kind of interesting that works because it seems that VolView is requesting an archive of the study, and then it must be unzipping that and loading the study client side ??

**Reply by sscotti** [reply]
> > That seems to work quite nicely with the cross-hairs, etc. Just wondering what the "requirements" are for uploaded series, etc.
> 
> I'm not sure what you refer to when you say "requirements". Could you please specify what you are looking for here?

This might also be related.  I had to write a little script to calculate the plane is some cases:

It logs the calculations like this.  For series that are 'glued' together by the tech it results in different values for that max_component.  Are you somehow splitting the series based on something like that ?

<img width="431" alt="image" src="https://user-images.githubusercontent.com/236933/233853601-99873c5b-c0aa-458b-8922-366225ea6dbb.png">

Just an example, not for that study:

INSTANCE|EDIT_SeriesDescription|{"max_component": 0.8065778069717093, "plane": "Sagittal"}
2023-04-21 23:54:48,546 | root | INFO     | 
INSTANCE|EDIT_SeriesDescription|{"max_component": 0.8730485107634399, "plane": "Sagittal"}

Selectively loading the other MPR's, e.g. Sagittal seems to work fine:

<img width="1001" alt="image" src="https://user-images.githubusercontent.com/236933/233855594-caef757e-b92e-46a2-a0bf-f2853a9373d1.png">

So, how does it select which one to display by default ?  When I load the entire study it just shows the Axials.
